tests/docker/Dockerfile

```diff
@@ -59,6 +60,15 @@ RUN mkdir -p "/opt/kafka-2.2.2" && chmod a+rw /opt/kafka-2.2.2 && curl -s "$KAFK
 RUN mkdir -p "/opt/kafka-2.3.1" && chmod a+rw /opt/kafka-2.3.1 && curl -s "$KAFKA_MIRROR/kafka_2.12-2.3.1.tgz" | tar xz --strip-components=1 -C "/opt/kafka-2.3.1"
 RUN mkdir -p "/opt/kafka-2.4.0" && chmod a+rw /opt/kafka-2.4.0 && curl -s "$KAFKA_MIRROR/kafka_2.12-2.4.0.tgz" | tar xz --strip-components=1 -C "/opt/kafka-2.4.0"
 
+# Install jars missing on JDK11.
+ARG MVN_MIRROR="https://repo1.maven.org/maven2"
+RUN if [ "x$jdk_version" = "xopenjdk:11" ] ; then curl -s "$MVN_MIRROR/com/sun/xml/bind/jaxb-impl/2.3.1/jaxb-impl-2.3.1.jar" -o /opt/kafka-0.10.1.1/libs/jaxb-impl-2.3.1.jar ; fi
+RUN if [ "x$jdk_version" = "xopenjdk:11" ] ; then curl -s "$MVN_MIRROR/com/sun/xml/bind/jaxb-impl/2.3.1/jaxb-impl-2.3.1.jar" -o /opt/kafka-0.10.2.2/libs/jaxb-impl-2.3.1.jar ; fi
+RUN if [ "x$jdk_version" = "xopenjdk:11" ] ; then curl -s "$MVN_MIRROR/com/sun/xml/bind/jaxb-impl/2.3.1/jaxb-impl-2.3.1.jar" -o /opt/kafka-0.11.0.3/libs/jaxb-impl-2.3.1.jar ; fi
+RUN if [ "x$jdk_version" = "xopenjdk:11" ] ; then curl -s "$MVN_MIRROR/javax/xml/bind/jaxb-api/2.3.1/jaxb-api-2.3.1.jar" -o /opt/kafka-0.10.1.1/libs/jaxb-api-2.3.1.jar ; fi
+RUN if [ "x$jdk_version" = "xopenjdk:11" ] ; then curl -s "$MVN_MIRROR/javax/xml/bind/jaxb-api/2.3.1/jaxb-api-2.3.1.jar" -o /opt/kafka-0.10.2.2/libs/jaxb-api-2.3.1.jar ; fi
+RUN if [ "x$jdk_version" = "xopenjdk:11" ] ; then curl -s "$MVN_MIRROR/javax/xml/bind/jaxb-api/2.3.1/jaxb-api-2.3.1.jar" -o /opt/kafka-0.11.0.3/libs/jaxb-api-2.3.1.jar ; fi
```

Comment:
I was thinking about this. It seems a bit weird to change the classpath in the system tests. There are 2 alternatives:

1. Change the older branches to include this jar in the classpath. It's a bit weird because the older versions don't support Java 11.
2. Change the compatibility tests to exclude the older versions when executed with Java 11.

Option 2 seems better to me. What do you think?

Reply:
> It seems a bit weird to change the classpath in the system tests 

I tend to agree with you in general but disagree in this specific case.

It's a well-known fact that JAXB(and some other libs) was removed from the JDK11.
It seems OK for me to explicitly add it to the classpath of the software that was developed when this lib was part of the JDK.

Please, see the link - https://www.oracle.com/technetwork/java/javase/11-relnote-issues-5012449.html#JDK-8190378

> Change the compatibility tests to exclude the older versions when executed with Java 11.

Do we need to check `upgrade_test.py` for 0.10 and 0.11?
I prepared the fix with the assumption that we still want to check all versions and upgrade scenarios. 

Please, note, that 0.8 and 0.9 cases succeed without classpath fix.

It seems inconsistent for me if we will disable 0.10 and 0.11 cases but keep 0.9 and 0.8 checks enabled. Do you suggest disabling all upgrade checks prior 1.x version?

Please, let me know what do you think and I will prepare the fix.

Reply:
> It's a well-known fact that JAXB(and some other libs) was removed from the JDK11.
> It seems OK for me to explicitly add it to the classpath of the software that was developed when this lib was part of the JDK.

I know, but my point is a bit different: if an old Kafka version is not tested with Java 11 as part of the unit and integration tests, it's a bit odd to expect the system tests to work with Java 11 for such version.

> It seems inconsistent for me if we will disable 0.10 and 0.11 cases but keep 0.9 and 0.8 checks enabled. Do you suggest disabling all upgrade checks prior 1.x version?

Yes, 1.0 and newer seems reasonable to me.

Reply:
Hello @ijuma 

I've updated PR.
Upgrade tests for old versions of Kafka on JDK 11 are disabled.

Upgrade tests results - https://issues.apache.org/jira/browse/KAFKA-9573?focusedCommentId=17060926&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-17060926

```
====================================================================================================
SESSION REPORT (ALL TESTS)
ducktape version: 0.7.6
session_id:       2020-03-17--013
run time:         52 minutes 37.462 seconds
tests run:        31
passed:           30
failed:           1
ignored:          0
====================================================================================================

```